ui-components/src/forms/HouseholdMemberForm.tsx

```diff
@@ -2,11 +2,19 @@ import React from "react"
 import { useRouter } from "next/router"
 import { HouseholdMemberUpdate } from "@bloom-housing/backend-core/types"
 import { t } from "../helpers/translator"
+import { AppearanceStyleType } from "../global/AppearanceTypes"
+import { Icon } from "../icons/Icon"
 import { ViewItem } from "../blocks/ViewItem"
 
-const HouseholdMemberForm = (props: { member: HouseholdMemberUpdate; type: string }) => {
+const HouseholdMemberForm = (props: {
+  member: HouseholdMemberUpdate
+  type: string
+  editMode?: boolean
+}) => {
   const { member, type } = props
   const router = useRouter()
+  let editMode = props.editMode
```

Comment:
Can we just use a default prop value here?

Reply:
Man, this must have been some weird debugging thing. Bad pattern! 😬 Will fix…